client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/transaction/XaModeTransactionTable.java

```diff
@@ -310,10 +316,10 @@ private void internalCommit() throws XAException {
          // commits only.
       }
 
-      private int internalPrepare(boolean onePhaseCommit) throws XAException {
+      private int internalPrepare() throws XAException {
          boolean readOnly = true;
          for (TransactionContext<?, ?> ctx : registeredCaches.values()) {
-            switch (ctx.prepareContext(currentXid, onePhaseCommit)) {
+            switch (ctx.prepareContext(currentXid, false)) {
```

Comment:
It shouldn't hold this up, but it looks like we can just remove the `onePhaseCommit` argument to `TransactionContext.prepareCommit`?